pypeit/spectrographs/keck_kcwi.py

```diff
@@ -38,6 +41,8 @@ def __init__(self):
         self.grating = None
         self.optical_model = None
         self.detector_map = None
+        self.slicescale = 0.00037718  # Degrees per 'large slicer' slice
```

Comment:
Can these be read from the Header?  And/Or should they be set dynamically inside `config_par`?